CI_YAML.md

```diff
@@ -70,6 +70,12 @@ targets:
           [
               {"dependency": "open_jdk", "version": "11"}
           ]
+          # Some dependencies are large, and stored in a cache for reuse
```

Comment:
out of curiosity, has there been any issues with fetching from cipd directly?

Reply:
I'm not sure about Java, but for example xcode is ~30 GB where it needs to be cached. In general, static deps like should be cached for the devicelab bots